fossunited/www/grants/directory/index.py

```
@@ -0,0 +1,51 @@
+        GRANTS_DIR,
+        fields=["name", "route", "json_data", "last_updated", "modified"],
+        filters={"is_published": 1},
+        order_by="last_updated desc, modified desc",
+        limit_page_length=200,
+    )
+
+    items = []
+    for d in docs:
+        try:
+            data = json.loads(d.get("json_data") or "{}")
+        except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)